apps/bsd/src/AppRoot.tsx

```diff
@@ -515,8 +454,7 @@ const App: React.FC = () => {
               <ExportResultsModal
                 onClose={() => setIsExportingCVRs(false)}
                 usbDriveStatus={displayUsbStatus}
-                election={election}
-                electionHash={electionHash}
+                electionDefinition={electionDefinition}
```

Comment:
Doesn't need to be in this PR but I've been thinking it might make more sense to move this to AppContext instead of passing through the components, curious what you think.

Reply:
That might make sense, though I think I'd be inclined to keep it as-is to make it easier to support multiple concurrent elections.